tests/helpers.py

```diff
@@ -1907,3 +1908,27 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+
+def skipif_ocs_version(expr):
+    """
+    This is a decorator to skip the test if condition evaluates to
+    true based on expression
+
+    Args:
+        expr (str): condition for which we need to check,
+            eg: '<4.3', '>4.3'
```

Comment:
add also example for `>=` .

What about if we will need to do something like . VERSION >=4.3 and VERSION < 4.6.

I can imagine that you can allow do something like  `> = 4.3 and {version} < 4.6` if you will do before expr.format(version=config.ENV_DATA['ocs_version'])

Reply:
@petr-balogh ,

What would be the best way to specify ranges during decoration in tests, 
how does this looks
```python
@skipif_ocsversion(">4.2 and <= 4.3")
def test_xyz()
    pass
```
is this ok ? or any other suggestions.

Reply:
Another possible syntax might be, to expect string or list of strings containing the simple expression like what it is documented (`<4.3`) - and then simply iterate over the list and ensure, that all the expressions are true... Not sure, which option is more clear and readable though.

Reply:
I agree with @dahorak's suggestion. I think we can start off iterating over a list of expressions (essentially only using `and`). This approach is readable and makes it clear to anyone reading this code what versions the test supports. If we feel the need for more operators like `or` we can revisit the logic.

Reply:
@dahorak @clacroix12 
sounds like good idea, it would be easy to iterate over list and build expression to eval.

Reply:
Done, user can pass a list